docs/content/preview/deploy/kubernetes/single-zone/gke/statefulset-yaml.md

```diff
@@ -64,12 +63,18 @@ Before starting deployment, perform the following:
 
 ## Create a GKE cluster
 
-Create a Kubernetes cluster by running the following command.
+Create a Kubernetes cluster using the following command:
 
 ```sh
 gcloud container clusters create yugabyte
```

Comment:
Why not just use the secure command - why even provide the public example?

Reply:
cc @nbhatia-yb @Arpit-yb

Reply:
I think instead of using "yugabyte", we should modify the command to - 
`gcloud container clusters create [CLUSTER_NAME]`

Reply:
Still think we should just drop this insecure example and only provide the secure one that follows it. So

`gcloud container clusters create cluster_name --enable-private-nodes`